mod/blog/languages/fi.php

```diff
@@ -42,8 +42,8 @@
 	'blog:error:revision_not_found' => 'Versiota ei löydy.',
 
 	// river
-	'river:create:object:blog' => '%s julkaisi blogiviestin %s',
-	'river:comment:object:blog' => '%s kommentoi blogiviestiä %s',
+	'river:object:blog:create' => '%s published a blog post %s',
+	'river:object:blog:comment' => '%s commented on the blog %s',
```

Comment:
Some of the river related translations are in English because the commit https://github.com/Elgg/Elgg/commit/1c91f7169d52d41684f28aa7dda7d041ec9b98e6#diff-32d69f3a43632b97f0b88383303787a0 changed to translation keys. These will get fixed once translators have made translations for the new keys.

Otherwise I do not see anything wrong in the translations.

Reply:
are translations pulled in english? why are not only translated strings pulled?

Reply:
This is the default behavior of the Transifex client. Presumably because all systems don't necessarily have a fallback system like Elgg has.

Reply:
well that is very problematic as local translations files will not fallback anymore, causing al kinds of issues... i believe this was not the case in earlier pulls am i right? Maybe there is an option to control this behaviour?

Reply:
Yes, there an option to exclude the untranslated strings. I can do that change.

What do you mean by local translation files? What exactly are the issues caused by this? AFAIK the pulls from Transifex have always worked like this.

Reply:
I checked, and yes, Elgg 2 also has English in place of the non-translated strings.

Reply:
that is strange... do you mean in the pull or in the repo... in the repo non translated strings are not present AFAICT... would mean we have 100% translated for all languages...

if all untranslated strings are present with their defaults in other than `en` languages this would mean that if i translated Englisch texts i would also need to change all translations in the other languages that use the same english translation

Reply:
In the repository. For example:
 -  https://github.com/Elgg/Elgg/blob/2.3/mod/groups/languages/da.php#L30-L33
 - https://github.com/Elgg/Elgg/blob/2.3/mod/groups/languages/el.php#L32-L33
 - https://github.com/Elgg/Elgg/blob/2.3/mod/groups/languages/sr.php#L32-L33

Reply:
i wonder what went wrong in that PR https://github.com/Elgg/Elgg/commit/8e16ee6e5430c5c26af223cb48e643adb930be5c... those translations should not have changed i think.. but doesn't matter, i would not pull in untranslated strings period

Reply:
Agree that we shouldn't pulled untranslated strings

Reply:
I just found out that the `--force` option in the `tx pull` command interferes with our goal. It causes also non-translated strings to be included in the files even when explicitly told not to.

I'm investigating whether we could remove the option.